src/services/school/index.js

```diff
@@ -23,6 +24,8 @@ module.exports = function schoolServices() {
 		lean: {
 			virtuals: true,
 		},
+		multi: true,
+		whitelist: defaultWhitelist,
```

Comment:
unfortunatelly, SHD has a text filter to search by school name, which breaks unless we add 
regexWhitelist and populateWhitelist (which is not exported) or simply use defaultModelServiceWhitelist instead.
However, the schools endpoint is big security concern, because IT IS OPENED